packages/mroonga-package-task.rb

```diff
@@ -11,9 +11,14 @@
 require "#{groonga_repository}/packages/packages-groonga-org-package-task"
 
 class MroongaPackageTask < PackagesGroongaOrgPackageTask
-  def initialize(mysql_package)
-    @mysql_package = mysql_package
-    super("#{@mysql_package}-mroonga", detect_version, detect_release_time)
+  def initialize(package_prefix)
+    if package_prefix == ""
+      @source_package = "mroonga"
+      super("#{@source_package}", detect_version, detect_release_time)
+    else
+      @mysql_package = package_prefix
+      super("#{@mysql_package}-mroonga", detect_version, detect_release_time)
+    end
```

Comment:
How about specifying full package name instead of MySQL package name?

```diff
diff --git a/packages/mroonga-package-task.rb b/packages/mroonga-package-task.rb
index e435a320..3871d4b4 100644
--- a/packages/mroonga-package-task.rb
+++ b/packages/mroonga-package-task.rb
@@ -11,9 +11,13 @@ end
 require "#{groonga_repository}/packages/packages-groonga-org-package-task"
 
 class MroongaPackageTask < PackagesGroongaOrgPackageTask
-  def initialize(mysql_package)
-    @mysql_package = mysql_package
-    super("#{@mysql_package}-mroonga", detect_version, detect_release_time)
+  def initialize(package_name)
+    super(package_name, detect_version, detect_release_time)
+    if package_name.end_with?("-mroonga")
+      @mysql_package = package_name.gsub(/-mroonga\z/, "")
+    else
+      @mysql_package = nil
+    end
     @original_archive_base_name = "mroonga-#{@version}"
     @original_archive_name = "#{@original_archive_base_name}.tar.gz"
   end
```